api/src/main/java/com/microsoft/gctoolkit/event/zgc/ZGCCollectionType.java

```diff
@@ -12,11 +15,9 @@ public enum ZGCCollectionType {
     }
 
     public static ZGCCollectionType get(String label) {
-        for (ZGCCollectionType status : ZGCCollectionType.values()) {
-            if (status.collectionLabel.equalsIgnoreCase(label.trim())) {
-                return status;
-            }
-        }
-        throw new IllegalArgumentException("No matching ZGCCollectionType found for: " + label);
+        return Arrays.stream(ZGCCollectionType.class.getEnumConstants())
```

Comment:
I'll accept this but... what do you think about introducing "Unknown" and return that instead of returning null?